devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -14,7 +14,8 @@ import groovy.json.JsonOutput
  *
  */
 metadata {
-    definition (name: "Arrival Sensor HA", namespace: "smartthings", author: "SmartThings") {
+    definition (name: "Arrival Sensor HA", namespace: "smartthings", author: "SmartThings",
+            runLocally: true, minHubCoreVersion: '000.018.00200', executeCommandsLocally: true) {
```

Comment:
Since the latest round of testing was all done with 0.25.x I think we should update `minHubCoreVersion` to be on the safe side

Reply:
seems reasonable

Reply:
updated